### ISSUE TITLE
TypeError: utils.getRegisteredComponents is not a function

### ISSUE THREAD

**Issue by blowsie** [vuejs/eslint-plugin-vue#715]
**Tell us about your environment**	
* **ESLint version:**	^5.9.0
* **eslint-plugin-vue version:** ^5.0.0	
* **Node version:**	v10.10.0

**Please show your full configuration:**

```javascript
module.exports = {
  root: true,
  env: {
    browser: true,
    node: true
  },
  parserOptions: {
    parser: 'babel-eslint'
  },
  extends: ['plugin:vue/recommended', 'plugin:prettier/recommended'],
  // required to lint *.vue files
  plugins: ['prettier', 'vue'],
  // add your custom rules here
  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
    // override/add rules settings here, such as:
    // 'vue/no-unused-vars': 'error'
  }
}

```

**What did you do?**	

```

eslint file.vue

Problem with ESLint reformatting: default.vue: TypeError: utils.getRegisteredComponents is not a function
```

**What did you expect to happen?**
success

**What actually happened?**

error

**Comment by armano2** [comment]
`eslint-plugin-vue version: ^5.9.0,` are you sure about this?

**Comment by blowsie** [comment]
> @blowsie `eslint-plugin-vue version: ^5.9.0,` are you sure about this? we just released `5.0.0`

Sorry this was a copy paste error, updated.

**Comment by blowsie** [comment]
For some reason updating my eslint settings in webstorm resolved this.
